File: Java/Shop/Start.java
import java.util.Scanner;

public class Start {
    public static void main(String[] args) {
        int ch;
        StockItem s = new StockItem(); //object reference
        Customer c = new Customer();  //object reference
        Scanner sc = new Scanner(System.in);
        do {
            System.out.println();
            System.out.println("-------------- MAIN MENU ---------------------");
            System.out.println("1. Stock Item");
            System.out.println("2. Customer");
            System.out.println("3. Exit");
            System.out.println("Enter choice: ");
            ch = sc.nextInt();
            switch (ch) {
                case 1:
                    s.menu();
                    break;
                case 2:
                    c.menu();
                    break;
                case 3:
                    break;
                default:
                    break;
            }
        } while (true);
        //scan.close();
    }
}

File: CPP/Shop/Start.cpp
#include <iostream>         //When our header files are in a pre-defined location
#include "StockItem.cpp"    //When our header files are in the current location
#include "Customer.cpp"

using namespace std;
int main(int argc, char const *argv[])
{
    int ch;
    stock_item *s = new stock_item();  //pointer to object
    customer *c = new customer();    //pointer to object
    do
    {
        cout<<endl;
        cout << "-------------- MAIN MENU ---------------------" << endl;
        cout << "1. Stock Item" << endl;
        cout << "2. Customer" << endl;
        cout << "3. Exit" << endl;
        cout << "Enter choice: ";
        cin >> ch;
        switch (ch)
        {
        case 1:
            s->menu();
            break;
        case 2:
            c->menu();
            break;
        case 3:
            exit(0);
            break;
        default:
            break;
        }
    } while (1);
    return 0;
}